persistence/play-jdbc-evolutions/src/main/scala/play/api/db/evolutions/Evolutions.scala

```diff
@@ -201,16 +201,31 @@ object Evolutions {
    * @param autocommit Whether to use autocommit or not, evolutions will be manually committed if false.
    * @param schema The schema where all the play evolution tables are saved in
```

Comment:
```suggestion
   * @param schema The schema that all the play evolution tables are saved in
```

I know this is the original scaladoc, but it's either "The schema that all the tables are saved in" or "The schema where the tables are saved" -- there's no dangling "in" when you use "where" and as an English student this makes me sad.